Gopkg.lock

```diff
@@ -1202,7 +1210,6 @@
   ]
   pruneopts = ""
   revision = "a2144134853fc9a27a7b1e3eb4f19f1a76df13c9"
-  source = "https://github.com/golang/crypto.git"
```

Comment:
I have added the source repo to Gopkg.toml, this should prevent these from being removed by `dep ensure` if you rebase.